Dockerfile

```diff
@@ -0,0 +1,43 @@
+FROM ubuntu:16.04
```

Comment:
If you use `FROM node:latest` you get way smaller images because you don't need to run apt-get (also, builds much faster)